stampifier.py

```diff
@@ -80,31 +74,21 @@ def get_stampified_content(self):
         # classify the page as stampifiable or not
         self._classify()
 
-        # early return when the pages are not
-        # stampifiable
-        if not self.is_stampifiable:
-            return {
-                "is_stampifiable": self.is_stampifiable,
-                "stamp_pages": None
-            }
-
         # summarize
         self._summarize()
+
         # order the stamp pages
         self._order_stamp_pages()
 
-        return {
-            "is_stampifiable": self.is_stampifiable,
-            "stamp_pages": self.stampified_pages
-        }
+        return self.stampified_pages
 
     def _classify(self):
         classifier = Classifier(
             self.preprocessed_contents,
             max_pages=self.max_pages,
             webpage_title=self._website.get_title()
         )
-        self.is_stampifiable = classifier.is_page_stampifiable()
+        classifier.is_page_stampifiable()
```

Comment:
classifier.classify()

Reply:
It makes sense to have `classify()` as a public function for classifier. You can merge/rename the methods in `classifier.py`

Reply:
sounds good! will update accordingly! :D